pyGAlib/gatools.py

```diff
@@ -396,21 +393,18 @@ def ExtractSubmatrix(adjmatrix, nodelist1, nodelist2=None):
 def SymmetriseMatrix(adjmatrix):
     """Converts a directed network into undirected by averaging the weights,
     i.e., bij = 1/2 * (aij + aji).
-
+ 
     Parameters
     ----------
     adjmatrix : ndarray of rank-2
         The adjacency matrix of the network.
-
+ 
     Returns
     -------
     An adjacency matrix of the same shape, of dype=float, with values
     """
-
-    if galib.Reciprocity(adjmatrix) == 1:
-        return adjmatrix
```

Comment:
I had to remove this because it was the only place that used the dependency to galib. Galib however uses gatools quite intensely. To me it looked as if the operations in `Reciprocity` are equally expensive anyway. Also,  the function now always returns a new matrix and not only sometimes, which might help avoiding subtle/hard-to-find bugs in the future.